python/gps/agent/mjc/agent_mjc.py

```diff
@@ -42,39 +43,55 @@ def _setup_world(self, filename):
         Args:
             filename: Path to XML file containing the world information.
         """
-        self._world = mjcpy.MJCWorld(filename)
+        self._world = []
+        self._model = []
 
-        self._model = self._world.get_model()
-        self._model = [copy.deepcopy(self._model)
-                       for _ in range(self._hyperparams['conditions'])]
+        # Initialize Mujoco worlds. If there's only one xml file, create a single world object,
+        # otherwise create a different world for each condition.
```

Comment:
Looks like this PR also adds functionality for different xml files per condition -- can you update the description of the PR?